stylelint-polaris/configs/coverage.js

```diff
@@ -14,14 +14,18 @@ module.exports = {
     'declaration-property-value-disallowed-list': [
       {
         display: ['grid', 'flex'],
+        // No numeric values
         top: [/\$.+/],
         bottom: [/\$.+/],
         left: [/\$.+/],
         right: [/\$.+/],
         width: [/\$.+/],
         height: [/\$.+/],
+        // zero and one only, no numeric values or additions
         opacity: [/(?!0|1)\d|[\d.]{2,}/],
+        // polaris custom properties only
```

Comment:
With this prompt, I think we need to update the regex: https://regex101.com/r/R8u8Ck/1